cppcache/integration-test/testThinClientTransactionsWithSticky.cpp

```diff
@@ -19,9 +19,6 @@
 
 DUNIT_MAIN
   {
-    CALL_TASK(Alter_Client_Grid_Property_1);
```

Comment:
Does removing this create a duplicate test where the same methods are being exercised already without grid set?

Reply:
No, just removes non-sense of toggling `grid-client`